.ci/docker/build.sh

```diff
@@ -199,6 +199,28 @@ case "$image" in
     CONDA_CMAKE=yes
     TRITON=yes
     ;;
+  pytorch-linux-focal-py3.12-clang10)
+    ANACONDA_PYTHON_VERSION=3.12
+    CLANG_VERSION=10
+    PROTOBUF=yes
+    DB=yes
+    VISION=yes
+    VULKAN_SDK_VERSION=1.2.162.1
+    SWIFTSHADER=yes
+    CONDA_CMAKE=yes
```

Comment:
The `cmake` version on Ubuntu 20.04 (focal) is 3.16.3 which is lowered than the lowest `cmake` version we support (3.18). We need to set `CONDA_CMAKE=yes`.

Reply:
It is safer to have a binary executable `cmake` in `PATH` rather than a Python wrapper.

The `cmake` package from PyPI will install a Python script `cmake` wraps on the true `cmake` binary in the wheel. This may have bugs in some situations.

- scikit-build/cmake-python-distributions#586
- pypa/pip#13222